app/src/main/java/com/architectcoders/arquitectomarvel/model/Repository.kt

```diff
@@ -20,6 +21,8 @@ class Repository(private val activity: Activity) {
         return MarvelApiRest.service.getCharacters(ts, publicKey, hash)
     }
 
+    val dao: ResultDao get() = ResultDatabase.getInstance(application).resultDao
```

Comment:
@jujoru Why this line has changed its position? Should it go to the top instead of in the middle?

Reply:
No, it does not affect where the instance is located. By bringing me your changes and merging it with mine, I would keep my instance that was above. If you see it better at the bottom or at the top I change it, no problem.

Reply:
I forgot to mention it! Yes, if you can move it to the top or the bottom it would be great :smiley: